M/Microbiome/Compat.toml

```diff
@@ -1,12 +1,12 @@
 ["0-0.4"]
-julia = "1.0.0-1"
+julia = "1.0"
 
 ["0.5-0"]
-julia = "1.1.0-1"
+julia = ["1.1", "1.2", "1.3"]
```

Comment:
I _think_ that these julia lines are not nessecery to change. Because "julia = 1.1.0" in a project toml [compat] (which get's written to registry as julia = "1.1.0-1", follows semver and so will include 1.1, 1.2, 1.3 and so un until 2.0, and all 1.0 versions of julia _should_ be compatible APIs and syntax. General only needed to update julia entries that included the 0.7 to 1.0 jump because 0.X type and 1.X type semvers behave differently.

Reply:
Alright, let's try it...